plugins/module_utils/network/sonic/config/lag_interfaces/lag_interfaces.py

```diff
@@ -638,15 +663,32 @@ def get_delete_ethernet_segment_requests(self, delete_members, want, have, delet
 
         return es_commands, es_requests
 
+    def get_delete_portchannel_request(self, name, attr):
+        url = f'data/sonic-portchannel:sonic-portchannel/PORTCHANNEL/PORTCHANNEL_LIST={name}'
+        if attr:
+            url += f'/{attr}'
```

Comment:
Please skip usage of 'format string', since it resulted in a pylint failure in python 2.7 during an earlier release in ansible automation hub. (Ref: https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/434)